assets/src/blocks/amp-story-post-title/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+import './edit.css';
+
+export const name = 'amp/amp-story-post-title';
+
+export const settings = {
+	title: __( 'Story Title', 'amp' ),
+	description: __( 'Displays the story title', 'amp' ),
```

Comment:
The descriptions from core blocks look to start with an imperative like “Display” and always end with a period.